Source/websocket/WebTransfer.h

```diff
@@ -508,7 +508,7 @@ namespace Web {
 
             // Read all Data to calculate the HASH/HMAC
             while (size > 0) {
-                uint16_t chunk = std::min(sizeof(buffer), size);
+                uint16_t chunk = std::min(sizeof(buffer), (size_t)size);
```

Comment:
Coding rule violation, no C-Style casting, only C++ style casting:
This should be: uint16_t chunk = std::min(static_cast<uint16_t>(sizeof(buffer)), size);